microcosm_pubsub/daemon.py

```diff
@@ -16,34 +14,35 @@ def make_arg_parser(self):
         parser.add_argument("--sqs-queue-url")
         return parser
 
-    @abstractproperty
```

Comment:
We no longer need to define schema mappings here. I haven't removed them yet, but I probably will eventually.